kordac/Kordac.py

```diff
@@ -2,22 +2,35 @@
 import mdx_math
 from kordac.KordacExtension import KordacExtension
 
-class Kordac():
+class Kordac(object):
+    """A converter object for converting markdown
+    to HTML"""
 
-    def run(self, md_string):
-        self.heading = 'I am a heading'
+    def run(self, markdown_string):
```

Comment:
Which one do you prefer:

- `html` and `html_tree`
- `html_string` and `html`